UnleashClient/constants.py

```diff
@@ -6,7 +6,7 @@
 REQUEST_TIMEOUT = 30
 REQUEST_RETRIES = 3
 METRIC_LAST_SENT_TIME = "mlst"
-CLIENT_SPEC_VERSION = "4.2.2"
+CLIENT_SPEC_VERSION = "4.3.0"
```

Comment:
Good suggestion by @sjaanus, just making this easier to do

```suggestion
CLIENT_SPEC_VERSION = "4.3.1"
```